src/parts/unspoil.ts

```diff
@@ -40,6 +42,7 @@ unspoil.command('unspoil', async ctx => {
             }
         )
     }
+    if (reply.text) {await ctx.reply(reply.from() + 'пишет: ||' + reply.text + '||')}
```

Comment:
1. Fix formatting
2. Reply from is not a text, its object (and getter)
3. For spoilers in text you need use parse mode flavour (see in static.ts and it's derivative) or you need explicitly set parse mode